e2e_tests/tests/experiment/experiment.py

```diff
@@ -615,15 +644,16 @@ def __str__(self) -> str:
 
 
 def get_experiment_durations(experiment_id: int, trial_idx: int) -> ExperimentDurations:
-    experiment_metadata = experiment_json(experiment_id)
+    r = bindings.get_GetExperiment(test_session(), experimentId=experiment_id)
```

Comment:
why not use the bindings for this one too?

Reply:
This is never called. I am going to remove this method and the ExperimentDurations class